### PR TITLE
Dynamic Background Animation

### PR DESCRIPTION
**Pull request by anojht**
Implemented a background fade in/fade out on a ten second interval to cycle backgrounds on landing and login pages.

### CODE REVIEW

**Comment by tidusjar** [comment]
The build is failing on some of the linting tasks. 

ClientApp/app/settings/settings.module.ts[143, 32]: file should end with a newline

**Comment by anojht** [comment]
@tidusjar Yes I think this is because the file in the develop branch does not end with a newline. Just double checked the branch from my fork and it's not causing the linting error.

**Comment by tidusjar** [comment]
My bad! i'll take a look

File: src/Ombi/ClientApp/app/landingpage/landingpage.component.ts [hunk 1]
```diff
@@ -41,4 +49,22 @@ export class LandingPageComponent implements OnInit {
             this.mediaServerStatus = x;
         });
     }
+
+    public ngOnDestroy() {
+        clearInterval(this.timer);
+    }
+
+    public cycleBackground() {
+        setTimeout(() => {
```
Comment (tidusjar) [inline src/Ombi/ClientApp/app/landingpage/landingpage.component.ts]: So the `this.timer` calls `this.cycleBackground` every 10 seconds, and then we have another two timers in here?

Once that clears the background and another that sets it. Why do we have this?

**Review by tidusjar — CHANGES_REQUESTED** [review]
See comments

File: src/Ombi/ClientApp/app/landingpage/landingpage.component.ts (hunk 1 above)
Comment (anojht) [reply src/Ombi/ClientApp/app/landingpage/landingpage.component.ts]: Previously I was running into sync issues therefore I added the timeout timers. With the new improvements I made they are redundant, will quickly push a new commit with the timers removed.

**Review by anojht — COMMENTED** [review]
Addressed the review request and made necessary changes

**Review by tidusjar — APPROVED** [review]

**Comment by tidusjar** [comment]
Nice!

**Pull request merged**